relayer/src/ethereum_checkpoints/utils/mod.rs

```diff
@@ -68,28 +31,16 @@ pub struct BootstrapResponse {
     pub data: Bootstrap,
 }
 
-pub fn deserialize_header<'de, D>(deserializer: D) -> Result<BeaconBlockHeader, D::Error>
```

Comment:
I don't see why we need it in the `ethereum-common` as this logic is clearly used only here and if this fact will change in future I think it'll be better to move `deserialize_header` along with `Bootstrap`, `FinalityUpdate` and `Update`

Reply:
ok, will move all these structs too